### ISSUE TITLE
SVG tags

### ISSUE THREAD

**Issue by penn5** [Kotlin/kotlinx.html#144]
None of the SVG tags appear to be present except for the root <svg> tag. For example, there is no <path> or <circle> tag

**Comment by spand** [comment]
Seems this is more suited for a sister library. This is a library for HTML, not general SVG/XML library.

**Comment by penn5** [comment]
Fair enough. Do you mind leaving this open to represent that it's a missing feature?

**Comment by spand** [comment]
I am not a maintainer, but looking at the issues list I am pretty sure it will stay :)
